src/assets/toolkit/scripts/toolkit.js

```diff
@@ -46,4 +47,20 @@ import 'prismjs/components/prism-scss';
     });
   });
 
+  var replyFormTemplate = document.getElementById('comment-reply-form-template');
+
+  if (replyFormTemplate) {
+    replyFormTemplate = replyFormTemplate.innerHTML;
+    arrayFromSelector('.js-comment').map(element => {
+      new CommentReply(element, {
+        formTemplate: replyFormTemplate,
+        onShow: CommentReply => {
+          Array.from(CommentReply.formElement.querySelectorAll('.js-FloatLabel')).map(element => {
```

Comment:
No, because I need to use `querySelectorAll` relative to `CommentReply.formElement`.